recipe/recipe.yaml

```diff
@@ -2,42 +2,40 @@
 schema_version: 1
 
 context:
-  version: 2.48.0
+  version: 2.49.0
 
 recipe:
   name: ray-packages
   version: ${{ version }}
 
 source:
   url: https://github.com/ray-project/ray/archive/ray-${{ version }}.tar.gz
-  sha256: 46321da3e619d2094055544b0e8c81765ed59820b94313c44a3c93290fe70cf0
+  sha256: cf9d2dbf6aae23fdbaca41297de115cd95daf81baf2005cd088cb3e0586a0763
   patches:
     - patches/0001-Disable-making-entry-scripts.patch
     - patches/0002-Ignore-warnings-in-event.cc-and-logging.cc.patch
-    - patches/0003-remove-dependencies.patch
+    - patches/0003-Newer-hermetic-python.patch
     # This needs modification for each release
-    - patches/0012-update-commit-sha.patch
-    - if: linux
-      then: patches/0010-cython-shebang.patch
+    - patches/0004-update-commit-sha.patch
     # See https://github.com/conda-forge/ray-packages-feedstock/issues/136
     # Keep in sync with current or active migration of libgrpc to avoid
     # ABI breakage
-    - patches/0004-Vendor-grpc-1.67.1.patch
-    - patches/0005-remove-stdlib-libc-from-.bazelrc.patch
+    - patches/0005-Vendor-grpc-1.67.1.patch
+    - patches/0006-remove-stdlib-libc-from-.bazelrc.patch
     # Patch around non-const NAN on windows in protobuf 27.5
-    - patches/0006-patch-protobuf-use-of-nan-on-windows.patch
-    - patches/0007-patch-zlib-in-prometheus-cpp-and-boost.patch
+    - patches/0007-patch-protobuf-use-of-nan-on-windows.patch
+    - patches/0008-patch-zlib-in-prometheus-cpp-and-boost.patch
     # Add missing include in new logging code for different absl
-    - patches/0008-add-missing-include.patch
+    - patches/0009-add-missing-include.patch
     # Patch bundled fmtlib/fmt in spdlog for char8_type which is invalid on macOS
-    - patches/0009-patch-bundled-fmt-in-spdlog-for-invalid-char8_type.patch
+    - patches/0010-patch-bundled-fmt-in-spdlog-for-invalid-char8_type.patch
     # Avoid building openssl-1.1.1f and redis, not needed in the shipped package
     - patches/0011-do-not-build-redis-openssl.patch
+    # https://github.com/conda-forge/docker-images/issues/311
+    - patches/0012-use-unzip-from-env.patch  # [linux]
```

Comment:
This needs conversion to `if: then:`

Reply:
I'm experimenting with some fixes locally; I don't think we should be patching in this way - it should be possible to point to unzip installed to $PREFIX from conda-forge.